tasks/main.yml

```diff
@@ -5,12 +5,12 @@
 - name: add ansible ppa
   apt_repository:
     repo: 'ppa:ansible/ansible'
+  when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
```

Comment:
this looks fine, i originally only created this role for debian based systems but we can widen the support